karma.shared.js

```diff
@@ -44,5 +44,6 @@ module.exports = {
   webpack: newWebpackConfig,
   webpackServer: {
     noInfo: true,
+    quiet: true
```

Comment:
This is a super-nitpick but adding a trailing comma here would keep it consistent (though it's mostly our fault for not having style tests in the repo).